src/components/Payroll/PayrollList/PayrollListPresentation.tsx

```diff
@@ -24,105 +23,112 @@ export const PayrollListPresentation = ({
   payrolls,
   paySchedules,
 }: PayrollListPresentationProps) => {
-  const { Badge, Button, Text } = useComponentContext()
+  const { Badge, Button, Heading, Text } = useComponentContext()
   useI18n('Payroll.PayrollList')
   const { t } = useTranslation('Payroll.PayrollList')
   const { locale } = useLocale()
 
   return (
-    <DataTable
-      emptyState={() => (
-        <Flex flexDirection="column" alignItems="center" gap={24}>
-          <div className={styles.doneIcon}>✓</div>
-          <Text>{t('emptyState')}</Text>
+    <Flex flexDirection="column" gap={16}>
```

Comment:
Wrapped this in the same flex layout I've been using in Payroll History for consistent spacing